src/current/v23.1/migrate-from-postgres.md

```diff
@@ -6,254 +6,291 @@ keywords: copy
+CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and is largely compatible with PostgreSQL syntax. 
 
-If you are working with a table that must be indexed on sequential keys, consider using [hash-sharded indexes]({% link {{ page.version.version }}/hash-sharded-indexes.md %}). We recommend doing thorough performance testing with and without hash-sharded indexes to see which works best for your application.
+For syntax differences, refer to [Features that differ from PostgreSQL]({% link {{ page.version.version }}/postgresql-compatibility.md %}#features-that-differ-from-postgresql).
 
-For further information, see [Unique ID best practices]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#unique-id-best-practices) and [3 Basic Rules for Choosing Indexes](https://www.cockroachlabs.com/blog/how-to-choose-db-index-keys/).
+### Unsupported features
 
-## Step 1. Dump the PostgreSQL database
+The following PostgreSQL features do not yet exist in CockroachDB:
 
-There are several ways to dump data from PostgreSQL to be imported into CockroachDB:
+{% include {{page.version.version}}/sql/unsupported-postgres-features.md %}
 
-- [Dump the entire database](#dump-the-entire-database)
-- [Dump one table at a time](#dump-one-table-at-a-time)
+## Load PostgreSQL data
 
-The import will fail if the dump file contains functions or type definitions. In addition to calling [`pg_dump`][pgdump] as shown below, you may need to edit the dump file to remove functions and data types.
+You can use one of the following methods to migrate PostgreSQL data to CockroachDB:
 
-Also, note that CockroachDB's [`IMPORT`][import] does not support automatically importing data from PostgreSQL's non-public [schemas][pgschema]. As a workaround, you can edit the dump file to change the table and schema names in the `CREATE TABLE` statements.
+- {% include {{ page.version.version }}/migration/load-data-import-into.md %} 
 
-### Dump the entire database
+       {% include {{ page.version.version }}/misc/import-perf.md %}
 
-Most users will want to import their entire PostgreSQL database all at once, as shown below in [Import a full database dump](#import-a-full-database-dump).
+- {% include {{ page.version.version }}/migration/load-data-third-party.md %}
 
-To dump the entire database, run the [`pg_dump`][pgdump] command shown below.
+- {% include {{ page.version.version }}/migration/load-data-copy-from.md %}
 
-{% include_cached copy-clipboard.html %}
-~~~ shell
-$ pg_dump employees > /tmp/employees-full.sql
-~~~
+The [following example](#example-migrate-frenchtowns-to-cockroachdb) uses `IMPORT INTO` to perform the initial data load.
```

Comment:
nit: I don't think we need to link to the example since it's right after this (unless our style guide says otherwise?)